src/NServiceBus.Core/Notifications/Notification.cs

```diff
@@ -0,0 +1,6 @@
+namespace NServiceBus
+{
+    interface Notification
```

Comment:
`INotification` would be more in line with the convention (`IMessage`, `ICommand`, `IEvent`)?